docs/cli/README.md

```diff
@@ -0,0 +1,482 @@
+
+Create a new backup for the stack.
+
+Options:
+  --version   Show version number                                      [boolean]
+  --help      Show help                                                [boolean]
+  --uploads   Include uploads in the backup.
+              (--no-uploads to disable)                [boolean] [default: true]
+  --database  Include database in the backup.
+              (--no-database to disable)               [boolean] [default: true]
+  --resume    Log ID for resuming an existing deploy.                   [string]
+  --debug     Enable internal debugging information.  [boolean] [default: false]
+```
+
+For example, to create a database only backup, you would run the following command:
+
+```sh
+bin/altis-cli stack backup --database --no-downloads platform-test
+
+Starting backup for platform-test...
+(node:72569) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
+(Use `node --trace-deprecation ...` to show where the warning was created)
+Backup started, resume later with...
+  hm stack backup platform-test --resume platform-test/backups:2025-02-10-11-33-07
+
+Connected!
+Starting new backup.
+Testing databases connection.
+Running mysqldump. Database size: 917mb
+→ mysqldump: [Warning] Using a password on the command line interface can be insecure.
+→ -- Connecting to platform-test-89c846f8.cnwdi9lvvkax.eu-west-1.rds.amazonaws.com...
+→ -- Retrieving table structure for table wp_10_commentmeta...
+→ -- Sending SELECT query...
+
+...
+
+→ -- Retrieving rows...
+→ -- Disconnecting from platform-test-89c846f8.cnwdi9lvvkax.eu-west-1.rds.amazonaws.com...
+Writing tar file.
+Uploading backup to S3. (111.13 MB)
+Writing manifest file.
+✔ Complete!
+```
+
+If a backup is going to take a long time and in case it may time out, the command outputs the resume command to use to resume
+the same backup.
+
+```sh
+Backup started, resume later with...
+  hm stack backup platform-test --resume platform-test/backups:2025-02-10-11-33-07
+```
+
+### Stack backups
+
+This command lists all the backups saved against the named stack. For example:
+
+```sh
+$ bin/altis-cli stack backups platform-test
+⠋ Loading backups for platform-test…
+✔ Loading backups for platform-test…
+? Select backup to download: (Use arrow keys)
+❯ platform-test-2025-02-10-12-16-24 (111.13MB)
+  platform-test-2025-02-10-12-13-34 (111.13MB)
+  platform-test-2025-02-10-12-10-28 (111.13MB)
+  platform-test-2025-02-10-11-38-21 (111.13MB)
+  platform-test-2025-02-05-12-41-11 (65.86GB)
+  platform-test-2025-01-31-13-46-53 (111.07MB)
+  platform-test-2025-01-28-2-24-40 (111.04MB)
+(Move up and down to reveal more choices)
+```
+
+Use your arrow keys to scroll through the backup names. It also displays the backup size. Pressing enter will download the
+specified back up to your default `Downloads` directory.
+
+```
+Downloading to /Users/mikelittle/Downloads/platform-test-2025-02-10-12-16-24
```

Comment:
Might be best to remove your own name

Reply:
Yep. Done and modified the screenshot too.